shared/actions/json/chat2.json

```diff
@@ -262,8 +262,8 @@
     "sendToPendingConversation": {
       "users": "Array<string>",
       "sendingAction": [
-        "More.ReturnType<typeof createMessageSend>",
-        "More.ReturnType<typeof createAttachmentUpload>"
+        "MessageSendPayload",
```

Comment:
importing More still needed in this file?

Reply:
? it doesn't refer to more